test_runner/regress/test_tenants.py

```diff
@@ -157,9 +157,20 @@ def test_metrics_normal_work(neon_env_builder: NeonEnvBuilder):
         )
 
 
-def test_pageserver_metrics_removed_after_detach(neon_env_builder: NeonEnvBuilder):
+@pytest.mark.parametrize(
+    "remote_storage_kind",
+    [RemoteStorageKind.NOOP, RemoteStorageKind.LOCAL_FS, RemoteStorageKind.MOCK_S3],
+)
```

Comment:
I wonder if this is really required. Maybe with noop and mock_s3 but not all three? I assume that it's currently still behind that Option whether there is a remote or not, unsure if the implementation has advanced.

Reply:
We definitely need `NOOP` and some other impl. I guess `LOCAL_FS` vs `MOCK_S3` makes no idfference at the storage_sync2 level.
Which one should I remove, then?

Reply:
I cannot remember what is the difference between mock_s3 and local_fs, maybe @SomeoneToIgnore has some insights?

Reply:
mock_s3 is a Python lib that is started in a background: http://docs.getmoto.org/en/latest/docs/server_mode.html#non-python-sdk-s-server-mode

local_fs is just a FS directory somewhere